guides/doc-Monitoring_Project/topics/con_metrics-storage.adoc

```diff
@@ -0,0 +1,12 @@
+[id='metrics-retention_{context}']
```

Comment:
```suggestion
[id='Metrics-Data-Retention_{context}']
```

Reply:
I guess this applies to the whole PR? You suggest the `id`s to be with capital letters?

When looking at existing guides, I see a mixture of `lower-case-with-dashes` and `Capitalized_with_Underscores`, so this one should be `Metrics_Data_Retention_{context}`?

Reply:
Based on the [mod docs template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc), IDs should be all lowercase:

`[id="my-concept-module-a_{context}"]`

I'd be in favor of renaming the ID to match the title, but keeping it lowercase:

`[id='metrics-data-retention_{context}']`

Reply:
Keeping lower and uppercase flexible here as it is not consistent throughout, you can just match ID with the title as @asteflova mentioned.